india_compliance/gst_india/doctype/gst_uom_mapping/gst_uom_mapping.json

```diff
@@ -0,0 +1,44 @@
+{
+ "actions": [],
+ "autoname": "autoincrement",
+ "creation": "2022-11-15 19:15:06.551557",
+ "default_view": "List",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "uom",
+  "gst_uom"
+ ],
+ "fields": [
+  {
+   "fieldname": "uom",
+   "fieldtype": "Link",
+   "in_list_view": 1,
```

Comment:
Try unique here, could work as frontend validation.

Reply:
unique will simply throw the error 'UOM must be unique' on save. So no need for backend validations then.